config/vxlan.py

```diff
@@ -23,7 +25,9 @@ def add_vxlan(db, vxlan_name, src_ip):
 
     if ADHOC_VALIDATION:
```

Comment:
>ADHOC_VALIDATION [](http://example.com/codeflow?start=7&length=16)

The design of ADHOC_VALIDATION should be on-par with yang validation. Could you ensure this constrain is also added into sonic yang models?

Reply:
Added yang-models validation as well: https://github.com/sonic-net/sonic-buildimage/pull/20108